README.md

```diff
@@ -23,4 +23,4 @@ Music Providers:
 4. run `npm install -g gulp`
 5. run `gulp`
 6. run `fig up`
-7. **Linux:** Navigate to [http://localhost:3000](http://localhost:3000) **OSX:** Navigate to your boot2docker vm ip:3000 (get the ip with `boot2docker ip`)
+7. **Linux:** Navigate to [http://localhost:3000](http://localhost:3000) **OSX:** Navigate to [http://dev.ammo.io:3000](http://dev.ammo.io:3000)
```

Comment:
So this was pretty annoying actually. When I was coding Fb authentication I was having some issues. The main issue was that when you create an app in fb you give it a legit IP, URL, or `localhost`. However .... when we are developing in our docker containers in OSX we are not really hitting `localhost` we have to go to our `boot2docker ip` which is a local IP facebook won't allow us to use. So.... what I came up for now as a temp fix is just to use the `boot2docker ip` and set up a sub domain. 

When we are developing in Linux you are running docker locally so you can actually just use `localhost` without a problem.

**tl;dr** No code was changed, just our domain settings.